spec/features/admin/admin_adds_items_spec.rb

```diff
@@ -23,15 +23,18 @@
     fill_in "Title", with: "Horrorland II"
     fill_in "Description", with: "Double scary"
     fill_in "Price", with: 59.95
-    #boxes do not appear when unchecked.
+
+    within('#category-items') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.